docker/clone-and-preflight.ts

```diff
@@ -65,14 +65,28 @@ if (projectUsesPostgresql) {
   // Example script:
   // https://github.com/upleveled/preflight-test-project-next-js-passing/blob/e65717f6951b5336bb0bd83c15bbc99caa67ebe9/scripts/alpine-postgresql-setup-and-start.sh
   const postgresUid = Number((await execa`id -u postgres`).stdout);
-  await execa({
+  const postgresProcess = await execa({
     // postgres user, for initdb and pg_ctl
     uid: postgresUid,
     // Show output to simplify debugging
-    stdout: 'inherit',
+    stdout: 'pipe',
     stderr: 'inherit',
   })`bash ./scripts/alpine-postgresql-setup-and-start.sh`;
 
+  const preflightEnvironmentVariables = postgresProcess.stdout.match(
+    /PREFLIGHT_ENVIRONMENT_VARIABLES:\n(\[.*?\])/,
+  )?.[1];
+
+  if (preflightEnvironmentVariables) {
+    const environmentVariablesKeys: string[] = JSON.parse(
+      preflightEnvironmentVariables,
+    );
```

Comment:
If the environment variables are found, `JSON.parse` converts the string representation of an array into an array.

`'["NEXTAUTH_URL", "APP_SECRET_KEY"]'` to `["NEXTAUTH_URL", "APP_SECRET_KEY"]`